client-rest/src/environments/environment.ts

```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
     production: false,
+    apiBaseUrl: 'https://localhost:3000',
```

Comment:
This makes sense but maybe the apiUrl below it needs a comment as to where that is used instead of assembling a url from the base

Reply:
Thanks, cleaned these up instead